docs/introduction/how-to-work-with-money.md

```diff
@@ -0,0 +1,388 @@
+- `Money::multiply(int|string $multiplier) : Money`
+- `Money::divide(int|string $divisor, int $scale) : Money`
+
+For addition and subtraction, the other parameter has to be also a `Money` instance.
+For multiplication and division, the other parameter has to be an integer or a numeric string (as they are able to represent decimal numbers precisely), not a float.
+
+The scale (number of decimal places) of the result is assigned automatically to all operations except division, keeping the results as precise as possible.
+Results of a division may be inexpressible with a finite decimal (eg. 1 / 3 = 0.3333...), so it's up to the user to specify the requested scale.
+- scale of the result of `add` and `subtract` is the *maximal scale* of both money values
+- scale of the result of `multiply` is the *sum of scales* of both money values
+- scale of the result of `divide` must be *explicitly specified*, the last decimal place will be rounded to minimize the error
+
+*Note: The scale of the money amount is always preserved - `getAmount` will use all decimal places of its scale (eg. zero money with scale 6 would return `0.000000`).*
+
+### Rounding
+
+You may use `Money::round(int $scale) : Money` method that rounds the amount of money up to `$scale` decimal places, it rounds 0.5 away from zero (making 1.5 into 2 and -1.5 into -2).
+This behavior is consistent with `PHP_ROUND_HALF_UP` rounding mode.
+
+The scale of the result will always be equal to the provided `$scale`.
+
+### Comparing
+
+To compare two monetary values you have to use the object's methods instead of [comparison operators](http://php.net/manual/en/language.operators.comparison.php):
+
+- `Money::equals(Money $other) : bool` instead of `===` and `==`
+- `Money::isLessThan(Money $other) : bool` instead of `<`
+- `Money::isGreaterThan(Money $other) : bool` instead of `>`
+- `Money::isLessThanOrEqualTo(Money $other) : bool` instead of `<=`
+- `Money::isGreaterThanOrEqualTo(Money $other) : bool` instead of `>=`
+- `Money::compare(Money $other) : int` instead of the Spaceship operator `<=>`
+
+To compare a value with zero you may use the short-hand methods`isPositive`, `isNegative` or `isZero`.
+
+All methods compare the amounts of both objects counting all decimal places.
+
+## Money in Forms
+
+For the user input of monetary values use the `MoneyType` (see [Symfony docs](https://symfony.com/doc/3.4/reference/forms/types/money.html) for details).
+
+```php
+use Symfony\Component\Form\Extension\Core\Type\MoneyType;
+use Symfony\Component\Validator\Constraints\NotBlank;
+
+// ...
+
+$orderItemFormBuilder->add('priceWithVat', MoneyType::class, [
+    'scale' => 6,
+    'constraints' => [
+        new NotBlank(['message' => 'Please enter unit price with VAT']),
+    ],
+]);
+```
+
+The form type is configured with a model data transformer that converts the value into a `Money` object automatically (`\Shopsys\FrameworkBundle\Form\Transformers\NumericToMoneyTransformer`).
```

Comment:
I think that the `Tranformer` class takes too much space here. Please change it to `NumericToMoneyTransformer` with link to the code.

It would be helpful in my opition to show that the form type really returns the `Money` instance and takes as an input the `Money` instance by an example (but I have no idea how the example should look like :-D).

Reply:
Addressed in 199b37562